requests-for-comments/2025-06-10-geo-and-llms-txt.md

```diff
@@ -0,0 +1,116 @@
+
+## Okay, so what actually changes?
+
+In terms of creating content, not much. Content still has to be [great](https://newsletter.posthog.com/i/156101009/your-great-content-probably-sucks), and there's no easy hack for that.
+
+But where things *do* change is in distribution and coverage. 
+
+In GEO, our primary goal is to have our brand, content, or insights included in the AI's synthesized response. This means we need more content in more places for the LLMs to retrieve and reference.
+
+For example, in the context GEO, it's more valuable for PostHog to appear across multiple spots in the top 20 Google results than to only be mentioned just once in the #1 spot. 
+
+This also implies more third-party content. Think:
+- Traditional PR and press (TechCrunch, Forbes, etc.)
+- Partnerships and syndication (e.g. comparison articles on dev.to or newsletters)
+- Youtube content
+- Community discussion on forums like Reddit
+
+![PR is important again 😂](https://res.cloudinary.com/dmukukwp6/image/upload/SCR_20250611_lqlo_3ca5da33be.png)
+
+## A note on training vs. retrieval
+
+Our content can appear in AI responses through either pathway: by being part of the model's training data or by being dynamically retrieved during a web search operation or other RAG methods. 
+
+GEO focuses on the second path, by influencing the LLM's diet instead of its training. In web retrieval, LLMs act more like reasoning engines, pulling from the web, distilling a bunch of information, and providing a structured answer with citations.
+
+## Some good reads
+- https://a16z.com/geo-over-seo/
+- https://www.mbcrosier.com/posts/ai-search-optimization
+- https://www.ignorance.ai/p/seo-for-ai-a-look-at-generative-engine
+
+## Solutions
+
+### llms.txt protocol
+I have a working branch. Submitting a PR this week.
+
+It's a [new standard](https://llmstxt.org/) that's been adopted by [Anthropic](https://docs.anthropic.com/llms.txt), [Cloudflare](https://developers.cloudflare.com/llms.txt), [MCP](https://modelcontextprotocol.io/llms.txt), [Cursor](https://docs.cursor.com/llms.txt), and others. We don’t know yet if the big LLM companies will officially treat it like `robots.txt`, but developers are already using it for context injection during retrieval.
+
+It basically turns all our public-facing content (docs, blog, and newsletters) into LLM-friendly sitemaps and markdown pages for AI crawling.
+
+### Targeting cited sources
+Tools like Gauge can show us the most cited sources for specific prompts. That gives us a clear target list, to either reach out and get mentioned or to create content on those same platforms.
+
+### Content distribution
+
+Open ended. Food for thought for content marketing.
+
+### Third-party content
+
+Open ended. Food for thought for content marketing.
```

Comment:
Beyond sponsorships, some newsletters like https://www.news.aakashg.com/ and https://read.technically.dev/ do paid content we could look into.